helm/chart/templates/keptnworkloadinstance-crd.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: keptnworkloadinstances.lifecycle.keptn.sh
   annotations:
     controller-gen.kubebuilder.io/version: v0.12.1
+    cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/klt-certs
```

Comment:
do we need '   '  or does it work the same?

Reply:
good catch, will adapt, but should not bother